lib/encryption.rb

```diff
@@ -0,0 +1,41 @@
+class Encryption
+  attr_reader :character_set
+
+  def initialize
+    @character_set = ("a".."z").to_a << " "
+  end
+
+  def indexed_alphabet
+    index_to_letter = @character_set.group_by do |letter|
+      @character_set.index(letter)
+    end
+
+    index_to_letter.transform_values! do |value|
+      value.join
```

Comment:
was assigning values as arrays containing a single letter of the alphabet. I wanted to be able to access them as string values, hence the transformation using the join method.